dev-tools/repos/python-tag

```diff
@@ -47,6 +47,7 @@ wazo-plugind-client
 wazo-provd
 wazo-provd-cli
 wazo-provd-client
+wazo-provd-plugins
```

Comment:
I'm not sure to understand why you need to add it here?
wazo-provd-plugins are released independently from wazo.
Then a wazo-20.01 can have the latest plugins

Reply:
That's one of the actions we decided in a grooming done on 2020-02-18... But I don't really remember our thinking about that and we did not write it in the document.